docs/users/blackfire-profiling.md

```diff
@@ -0,0 +1,25 @@
+## Profiling with Blackfire.io
+
+DDEV-Local has built-in [blackfire.io](https://blackfire.io) integration.
+
+### Basic Blackfire Usage
+
+1. Create an account on [blackfire.io](https://blackfire.io)
+2. Get the Server ID and the Server Token from your Account->Credentials on blackfire.io
+3. Configure ddev with the credentials, `ddev config global --web-environment="BLACKFIRE_SERVER_ID=<id>,BLACKFIRE_SERVER_TOKEN=<token>"`. It's easiest to do this globally, but you can do the same thing with project-level `ddev config`.
```

Comment:
I noticed that this command doesn't append but replaces the env vars to the global `web_environment` setting.
Is there a way to append variables instead?

Reply:
The environment variable feature was just added in the previous PR (prompted by the need here, you used to have to do a docker-compose override), so I'm sure it will have some maturing to do. Most people just edit the appropriate configuration, the project .ddev/config.yaml or the global ~/.ddev/global_config.yaml - so that's the easiest way to just "edit" a set of env vars.